resources/JS/script.js

```diff
@@ -42,3 +42,138 @@ window.onscroll = () => {
+    image: '<img src="./resources/images/desktop-2.png" alt="project-two" class="project-img hov-img" id="project-img2">',
+    popupImage: '<img src="./resources/images/desktop-2.png" alt="project" class="project-img" id="popup-img">',
+    technologies: ['html', 'css', 'javascript'],
+    live: '',
+    source: '',
+  },
+  {
+    name: 'Tonic',
+    mobileDesc: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    desktopDesc: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    image: '<img src="./resources/images/desktop-3.png" alt="project-three" class="project-img hov-img">',
+    popupImage: '<img src="./resources/images/desktop-3.png" alt="project" class="project-img" id="popup-img">',
+    technologies: ['html', 'css', 'javascript'],
+    live: '',
+    source: '',
+  },
+  {
+    name: 'Multi-Post Stories',
+    mobileDesc: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    desktopDesc: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    image: '<img src="./resources/images/desktop-4.png" alt="project-four" class="project-img hov-img" id="project-img4">',
+    popupImage: '<img src="./resources/images/desktop-4.png" alt="project" class="project-img" id="popup-img">',
+    technologies: ['html', 'css', 'javascript'],
+    live: '',
+    source: '',
+  },
+];
+
+// Select portfolio section of html document
+const portfolio = document.getElementById('portfolio');
+
+// Loop through each project and create a new project div and append it to the portfolio section
+projects.forEach((element) => {
+  const project = document.createElement('div');
+  project.className = 'project';
+  project.innerHTML = `
+    ${element.image}
+    <div>
+      <h2>${element.name}</h2>
+      <div class="project-type">
+        <h3 class="text-font">CANOPY</h3>
+        <img src="./resources/images/counter.svg" alt="counter" />
+        <h4 class="text-font">Back End Dev</h4>
+        <img src="./resources/images/counter.svg" alt="counter" />
+        <h4 class="text-font">2015</h4>
```

Comment:
- [ ] Kindly change the color of this text to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/195307816-28d4fe62-e787-48ba-ad5d-958e7340a88a.png) | ![image](https://user-images.githubusercontent.com/24703171/195307987-63044526-fdd2-4ecf-a8a9-da8382c7bfa8.png) |